### ISSUE TITLE
DST Staging Review: Selected state is duplicated with Android Talkback

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#4680]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** New Tabs Design VADS Component
**Staging Review ticket:** [Staging review: New Tabs Design VADS Component department-of-veterans-affairs/va.gov-team#115098](https://github.com/department-of-veterans-affairs/va.gov-team/issues/115098)

## Findings details
**VA.gov Experience Standard - issue:** [User encounters consistent, clear design components and patterns.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**This is an issue with the:** 
**High-priority:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Using Android, on Chrome, with Talkback on, when the user selects another tab, the first tab remains visually selected. 

<details><summary>Link, screenshot or steps to recreate</summary>

![Android Talkback tabs screenshot; selected state issue](https://github.com/user-attachments/assets/95cee5f9-02da-4fb2-ae89-8301553c1134)

</details>

## Recommended action
Ensure that the tab with the atribute`aria-selected="true"` is the only tab with the corresponding visual treatment that indicates selected. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation

**Comment by jeana-adhoc** [comment]
I'd like to see if @jasonday can re-test this using his same device [on this link](https://design.va.gov/storybook/iframe.html?globals=&args=&id=components-va-tabs--default&viewMode=story) I'm having a hard time recreating it, and others who have, can't recreate it either.   Based on the url in the screenshot, I wonder if an earlier, not final version, of tabs were tested.
